apps/sms/views/conversation/js/link_helper.js

```diff
@@ -150,6 +150,10 @@ var LINK_TYPES = {
         return false;
       }
 
+      if(url[0] === '.') {
```

Comment:
with this change, you miss to filter something like `http://.website.com` which is also incorrect.
So I'd suggest 2 possibilities:

* use `if (match[2][0] === '.')` (or maybe `match[2].startsWith('.')` is more readable).
* change the regexp for {2} to `'([\\w](?:\\[.][-\\w]){0,255})',`

I'd rather change the regexp (if this works properly ;) ). Can you add a test for "http://.website.com" too ? And test various cases in Firefox or on a device ?